UI/app/src/main/res/layout/activity_driver_dashboard.xml

```diff
@@ -330,10 +329,98 @@
 
         </RelativeLayout>
 
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="65dp">
+
+            <TextView
+                android:id="@+id/busLineName"
+                android:layout_width="78dp"
+                android:layout_height="29dp"
+                android:autoText="false"
+                android:clickable="false"
+                android:editable="false"
+                android:fontFamily="@font/basic"
+                android:gravity="center|center_horizontal"
+                android:importantForAutofill="no"
+                android:textColor="@color/yellow"
+                android:textSize="16dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.571" />
+
+            <TextView
+                android:id="@+id/nextStopView"
+                android:layout_width="203dp"
+                android:layout_height="31dp"
+                android:layout_marginStart="12dp"
+                android:layout_marginLeft="12dp"
+                android:clickable="true"
+                android:editable="false"
+                android:fontFamily="@font/basic"
+                android:gravity="center|center_vertical"
+                android:importantForAutofill="no"
+                android:onClick="displayAllStops"
+                android:text="@string/display_first_stop"
+                android:textColor="@color/yellow"
+                android:textSize="16dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/busLineName"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.571" />
+
+            <Button
+                android:id="@+id/changeStop"
+                android:layout_width="75dp"
+                android:layout_height="40dp"
+                android:onClick="changeStop"
+                android:text="@string/next"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.483"
+                app:layout_constraintStart_toEndOf="@+id/nextStopView"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.64" />
```

Comment:
Don't we already have a button for next stop?

Reply:
Mmmh, we have a layout that does not seem to be a button but more of an indicator of a passenger asking to stop at the next stop

Reply:
But if not, we can do so that this indicator is also a button, which is not a lot of changed in the logic =)

Reply:
You can set an onClick listener in the nextStop function I set up. It's been deleted though, so may have to put it back. Just for UX and UI, might be better to have one button for the same function :)